packages/core/src/Box/examples/BoxPolymorphicExample.tsx

```diff
@@ -7,15 +7,6 @@ export default function BoxSpacingExample() {
       <Box as="span" display="inline" marginRight="spacingL">
         I am span
       </Box>
-      <Box
-        as="a"
```

Comment:
Too controversial 😄. We shouldn’t recommend using Box to create links, users should use `TextLink`.

Reply:
Maybe we need to replace this example with something else to keep 3 items in a row?

Reply:
Why do we need 3 items?

Reply:
just to have more than 2 examples :)